md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/UsersAPI.java

```diff
@@ -63,15 +62,14 @@ RestWrapper get(
                 users.setUsername(jpaUsers.getUsername());
                 users.setPassword(jpaUsers.getPassword());
 
-                users.setEnabled((jpaUsers.getEnabled() == true) ? (short) 1 : 0);
+                users.setEnabled((jpaUsers.getEnabled()) ? (short) 1 : 0);
 
             }
-            //  users = s.selectOne("call_procedures.GetUsers", users);
-
             restWrapper = new RestWrapper(users, RestWrapper.OK);
             LOGGER.info("Record with name:" + username + " selected from Users by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with name:" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)